lib/cocoapods-binary-cache/pod-rome/xcodebuild_raw.rb

```diff
@@ -44,5 +50,47 @@ def self.xcodebuild(options)
         raise "Fail to build targets: #{targets}"
       end
     end
+
+    def self.xcodebuild_archive(options)
```

Comment:
can we reuse the xcodebuild arguments from the `def self.xcodebuild(options)` method?